Application/EdFi.Ods.AdminApi/appsettings.json

```diff
@@ -4,7 +4,8 @@
         "PathBase": "",
         "DefaultPageSizeOffset": 0,
         "DefaultPageSizeLimit": 25,
-        "MultiTenancy": false
+        "MultiTenancy": false,
+        "PreventDuplicateApplications": false
```

Comment:
I think we should include the "PreventDuplicateApplications" setting in the following compose files:
- [compose-build-binaries-multi-tenant.yml](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/blob/ADMINAPI-321/Docker/Compose/pgsql/MultiTenant/compose-build-binaries-multi-tenant.yml) 
- [compose-build-dev-multi-tenant.yml](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/blob/ADMINAPI-321/Docker/Compose/pgsql/MultiTenant/compose-build-dev-multi-tenant.yml)
- [compose-build-binaries.yml](https://github.com/Ed-Fi-Alliance-OSS/AdminAPI-2.x/blob/ADMINAPI-321/Docker/Compose/pgsql/SingleTenant/compose-build-binaries.yml)